addons/website_hr_recruitment/data/hr_job_demo.xml

```diff
@@ -198,33 +198,6 @@
                     </div>
                 </div>
             </section>
-            <!-- Photos -->
```

Comment:
to be replaced and not just removed

Reply:
Do you have something to suggest as a replacement ?
As it's just Demo data, are those pictures really useful for something ?
FYI, it was discussed with al that just asked to remove it.

Reply:
It fun like decision when you know that FP takes sevral weeks with @bvr-odoo  to make this page 'perfect pixel'.
I'm the first to think that the photos are personal, outdated and probably not to be shared for other companies...

But imo, just remove code to become debian compliant, is like remove your test that is red to have a green branch ;)
But if you already discuss about it and decided to just remove, it is not a problem, while it has been thoughtful... it was my point.